contracts/augur/trading/ZeroXTrade.sol

```diff
@@ -191,26 +48,31 @@ contract ZeroXTrade is Initializable, IZeroXTrade, IERC1155 {
         address _affiliateAddress,
         bytes32 _tradeGroupId,
         IExchange.Order[] memory _orders,
-        bytes[] memory _signatures
+        bytes[] memory _signatures,
+        address _taker
```

Comment:
Why does this take a `taker` param instead of using msg.sender? This would mean anyone could force someone else into being a taker.